test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.csproj

```diff
@@ -55,4 +49,8 @@
     <ProjectReference Include="..\..\..\src\Microsoft.Spatial\Microsoft.Spatial.csproj" />
 
   </ItemGroup>
+
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent" Condition="'$(TargetFramework)' == 'net452'">

```

Comment:
Register the strong name only needs once. changes here mean to register it every time?
besides, to register it needs to run the project under administrator?

Reply:
There's really no harm in running it every time, though if you know of a way to check that it's already been run I'm happy to incorporate it. 

As for admin, there are already projects that require admin privileges to build, so I don't really see an issue with requiring it for this since it allows one fewer step before running the tests.